jsonrpc/src/schemas/trace.json

```diff
@@ -0,0 +1,89 @@
+{
+  "traceResultObject": {
+    "title": "traceResultObject",
+    "description": "Trace data for the result of tracing content request.",
+    "type": "object",
+    "required": [
+      "origin",
+      "targetId",
+      "receivedFrom",
+      "responses",
+      "metadata"
+    ],
+    "properties": {
+      "origin": {
+        "description": "Local Node ID",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "targetId": {
+        "description": "Target content ID",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "receivedFrom": {
```

Comment:
Shouldn't this field be optional? Or at least we should specify what the value should be if no content was received.

Reply:
agreed